crxDownloader.user.js

```diff
@@ -40,14 +39,15 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
 For more information, please refer to <http://unlicense.org/>*/
+"use strict";
```

Comment:
Are you sure that this will work in chrome without wrapping into a closure? As I know, it won't. Will fix.